packages/backend/src/Api/Middleware/isRole.ts

```diff
@@ -0,0 +1,39 @@
+import { NextFunction, Request, Response } from "express";
+import AuthService from "Api/Modules/Client/Authentication/Services/AuthAccountService";
+// import SettingsUserRoleService from "Api/Modules/Client/OnboardingAndAuthentication/Services/SettingsUserRoleService";
```

Comment:
Can you please remove the commented code? Also, doesn't `isRole` return a boolean or something?